mono/metadata/appdomain.c

```diff
@@ -1448,6 +1449,7 @@ get_shadow_assembly_location_base (MonoDomain *domain, MonoError *error)
 	char *cache_path, *appname;
 	char *userdir;
 	char *location;
+	char *username;
```

Comment:
yes, the call to g_get_user_name leaks on windows and this was part of a different fix, will remove.